src/components/Comment/CommentInput/CommentInput.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react';
+import { useToast } from '@/contexts';
+import { postComment } from '@/api/comment';
+import { postCommentErrorMessage } from '@/utils/errorMessage';
+import formStyles from '@/styles/helpers/formHelpers.module.scss';
+import buttonStyles from '@/styles/helpers/buttonHelpers.module.scss';
+import styles from './CommentInput.module.scss';
+
+const CommentInput = ({ productId, refreshAfterSubmit }) => {
+  const { showToast } = useToast();
+  const [content, setContent] = useState('');
+
+  const handleSubmit = async () => {
+    if (!content.trim()) return;
```

Comment:
💊 제안
버튼이 비활성화 상태일때는 해당 함수가 실행되지 않으니 해당 조건문은 없어도 될 것 같아요!
```suggestion
  const handleSubmit = async () => {
```